apps/ui/src/networks/evm/actions.ts

```diff
@@ -613,6 +613,13 @@ export function createActions(
           functionParams: [delegatee],
           abi: ['function delegate(address delegatee)']
         };
+      } else if (delegationType === 'compound-governor') {
+        contractParams = {
+          address: delegationContract,
+          functionName: 'delegate',
+          functionParams: [delegatee],
+          abi: ['function delegate(address delegatee)']
+        };
```

Comment:
This needs to use same format as `governor-subgraph` (we need to split `delegationContract` to remove networkId),
we add it here:
https://github.com/snapshot-labs/sx-monorepo/blob/4bfba1c11252bdbab241677ca7734ec7f0828dbb/apps/ui/src/components/Modal/Delegate.vue#L66